src/sentry/templates/sentry/error-page-embed.js

```diff
@@ -87,25 +91,47 @@
       }
     }
 
-    var linkTags = this.element.getElementsByTagName('a');
-
-    var onclickHandler = function(e) {
+    var cancelHandler = function(e) {
       e.preventDefault();
       self.close();
     };
-
-    for (i = 0; i < linkTags.length; i++) {
-      if (linkTags[i].className === 'close') {
```

Comment:
nice cleanup, that's some wonky code

Reply:
Looks like a getElementsByClassName() from before that was a thing.

Reply:
Might have been a browser compatibly thing. Since this is a piece of code that needs to be highly compatible